sdk/forms/src/protected-view/iframe/components/view.ts

```diff
@@ -10,7 +10,7 @@ export const View = component(
     view.classList.add('view');
     const value = result.strategy === 'invoke-action' ? result.response : result.objects;
     const values = display.map(({ path, label, clickToCopy, class: className }) =>
-      DisplayValue({ value, path, label, clickToCopy, className, sendToParent }),
+      DisplayValue()({ value, path, label, clickToCopy, className, sendToParent }),
```

Comment:
Why changing it to a function that return component?

Reply:
Same component and reasoning as above